lib/package.gi

```diff
@@ -2778,83 +2780,96 @@ InstallGlobalFunction( BibEntry, function( arg )
     return entry;
 end );
 
-Unbind( Unicode );
-Unbind( Encode );
-
 # dummy assignments to functions to be read lated in the GAPDoc package
 ParseBibXMLextString:= "dummy";
 StringBibXMLEntry:= "dummy";
 
 InstallGlobalFunction( Cite, function(arg)
-local name, bib, key, parse, year;
-if Length(arg)=0 then 
-  name:="GAP";
-else
-  name := NormalizedWhitespace(arg[1]);
-fi;
-if name="gap" then 
-  name:="GAP"; 
-fi;
-if Length(arg)<=1 then
-  bib:= BibEntry( name );
-elif Length(arg)>2 then 
-  Error("`Cite' takes no more than two arguments");
-else
-  key:=arg[2];
-  bib:= BibEntry( name, key );
-fi;
-if bib="" then
-  Print("WARNING: No working version of package ", name, " is available!\n");
-  return;
-fi;
-parse:= ParseBibXMLextString( bib );
-Print("Please use one of the following samples\n",
-      "to cite ", name, " version from this installation\n\n");
+  local name, bib, key, parse, year, en, pkginfo;
+  if Length(arg)=0 then 
+    name:="GAP";
+  else
+    name := NormalizedWhitespace(arg[1]);
+  fi;
+  if LowercaseString(name) = "gap" then 
+    name:="GAP"; 
+  else
+    pkginfo := InstalledPackageVersion(name);
+    pkginfo := First(PackageInfo(name), r-> r.Version = pkginfo);
+    name := pkginfo.PackageName;
```

Comment:
This code is also new and not mentioned in the commit message resp. PR description as far as I can tell? My guess is that it's there to "normalize" the spelling (in terms of case) of the package name?